eng/config/PublishData.json

```diff
@@ -235,7 +247,7 @@
       "vsBranch": "main",
       "vsMajorVersion": 17,
       "insertionCreateDraftPR": false,
```

Comment:
Do we want to switch this to `true` for now so some of the unnecessary insertion checks are skipped?

Reply:
I think the checks would trigger as soon as I make them non-draft to insert them though, so would anything actually be skipped?  Or is there some other need to not run these checks on all prs